src/components/LicenseRevokeModal/index.jsx

```diff
@@ -115,16 +131,14 @@ class LicenseRevokeModal extends React.Component {
             <ul className="m-0 pl-4">
               {error.map(message => <li key={message}>{message}</li>)}
             </ul>
-          ) : (
-            error[0]
-          )}
+          ) : error[0]}
         />
       </div>
     );
   }
 
   renderTitle() {
-    return <small>Are you sure you want to revoke access?</small>;
+    return <small>Are you sure you want to revoke license?</small>;
```

Comment:
Nit: "Are you sure you want to revoke _this_ license?"

Reply:
I agree. I was just going off the copy provided in the wireframe 🤷‍♂️ I will add _this_.